pkg/kube/kube.go

```diff
@@ -104,6 +105,21 @@ func (k *KubeClient) Delete(manifests []byte, namespace string) ([]byte, error)
 	return out, nil
 }
 
+func (k *KubeClient) DeleteByName(name, kind, namespace string) ([]byte, error) {
+	args := []string{
+		"delete",
+		"--namespace", namespace,
+		kind, name,
+	}
+
+	out, err := k.runKubectlCmd(args)
```

Comment:
I have seen this on the other kube commands. Is there a reason we dont just return k.runKubectlCmd?

Reply:
Now that you mention it... no, not that I can see. I think I must have just copied the body and replaced the args. I'll fix that.